frontend/learning/components/CoreLearning/CoreLearning.js

```diff
@@ -74,9 +72,18 @@ export default function CoreLearning(props) {
         </>
     );
 
+    const showAssignmentPage = () => {
+        props.history.push(`/learning/assignment/${qid}`);
+    };
```

Comment:
I don't think we can use history if we are not using react router? Not sure about this tho